app/components/navpatientheader/DemographicInfo.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { withTranslation } from 'react-i18next';
+import { Flex, Text } from 'theme-ui';
+import moment from 'moment';
+
+const JS_DATE_FORMAT = 'YYYY-MM-DD';
+
+const DemographicsContainer = ({ children }) => (
+  <Flex sx={{
+    color: 'text.primary', 
+    flexShrink: 0, 
+    gap: 5, 
+    fontSize: 1, 
+    alignItems: 'flex-start', 
+    height: '1.5rem',
+    alignItems: 'center'
+  }}>
+    { children }
+  </Flex>
+)
```

Comment:
I hope you don't mind me using this pattern here or there. In my mind, when you read the returned JSX of a component like <DemographicInfo /> you really want to see how the data is flowing - data binding, event handling, etc - to understand how it ultimately derives the returned DOM elements. For me, abstracting out components that are heavy on styling helps enhance the visibility of those data flow elements so you can see what the component is doing.

Reply:
Sounds good, perhaps in one of our "FE huddles" :)